lib/components/search/SearchFilter.vue

```diff
@@ -40,11 +40,11 @@ const props = defineProps({
   },
 })
 
-const isActive = ref(props.activeFilters.value.includes(props.facetName))
+const isActive = ref(props.activeFilters.includes(props.facetName))
 const emit = defineEmits(['toggle'])
 
 watchEffect(() => {
-  isActive.value = props.activeFilters.value.includes(props.facetName)
+  isActive.value = props.activeFilters.includes(props.facetName)
```

Comment:
I don't see `isActive` be changed anywhere else. I think `isActive` should be a computed ref instead.

```js
const isActive = computed(() => props.activeFilters.includes(props.facetName))
```

Reply:
tbh, I didn't even know what `compute` and had to read the docs... 
but makes sense